integration-cli/docker_cli_tag_test.go

```diff
@@ -27,14 +28,11 @@ func (s *DockerSuite) TestTagUnprefixedRepoByID(c *check.C) {
 
 // ensure we don't allow the use of invalid repository names; these tag operations should fail
 func (s *DockerSuite) TestTagInvalidUnprefixedRepo(c *check.C) {
-
 	invalidRepos := []string{"fo$z$", "Foo@3cc", "Foo$3", "Foo*3", "Fo^3", "Foo!3", "F)xcz(", "fo%asd"}
 
 	for _, repo := range invalidRepos {
-		_, _, err := dockerCmdWithError("tag", "busybox", repo)
-		if err == nil {
-			c.Fatalf("tag busybox %v should have failed", repo)
-		}
+		out, _, err := dockerCmdWithError("tag", "busybox", repo)
+		c.Assert(err, check.NotNil, check.Commentf(out))
```

Comment:
I would have done something more like `c.Assert(err, check.NotNil, check.Commentf("tag busybox %v should have failed : %v", repo, out))`.